### PR TITLE
Upgrade Docker version in dind-aws image

### PR DESCRIPTION
**Pull request by mremi**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ Versions
 -----------
 
 * Add notes here
+* Upgrade Docker version in dind-aws image
```
Comment (rande) [inline CHANGELOG.md]: Can you can add the version ?

File: dind-aws/Dockerfile [hunk 2]
```diff
@@ -17,6 +19,24 @@ RUN echo "Install AWS Cli" && \
...
+        "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+        $(lsb_release -cs) \
+        stable" && \
+    apt-get -q update && \
+    apt-get -qy install docker-ce=${DOCKER_VERSION} && \
+    docker --version && \
+
+    echo "Update Docker Compose" && \
+    rm -f /usr/local/bin/docker-compose && \
+    curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` -o /usr/local/bin/docker-compose && \
```
Comment (fpavageau) [inline dind-aws/Dockerfile]: `curl -sSL` would be better than just `curl -L` : silent, yet errors are reported.

File: CHANGELOG.md (hunk 1 above)
Comment (mremi) [reply CHANGELOG.md]: done

File: dind-aws/Dockerfile (hunk 2 above)
Comment (mremi) [reply dind-aws/Dockerfile]: done

**Comment by fpavageau** [comment]
LGTM 👍

**Pull request merged**